chapters/data-analysis.tex

```diff
@@ -1,292 +1,487 @@
+
+% Version control
+Finally, everything that can be version-controlled should be. 
+Version control allows you to effectively track code edits,
+including the addition and deletion of files. 
+This way you can delete code you no longer need, 
+and still recover it easily if you ever need to get back previous work.
+Both analysis results and data sets will change with the code.
+You should have each of them stored with the code that created it.
+If you are writing code in Git/GitHub,
+you can output plain text files such as \texttt{.tex} tables
+and metadata saved in \texttt{.txt} or \texttt{.csv} to that directory.
+Binary files that compile the tables,
+as well as the complete data sets, on the other hand,
+should be stored in your team's shared folder. 
+Whenever data cleaning or data construction codes are edited,
+use the master script to run all the code for your project.
+Git will highlight the changes that were in data sets and results that they entail. 
+
+%------------------------------------------------
+
+\section{Data cleaning}
+
+% intro: what is data cleaning -------------------------------------------------
+Data cleaning is the first stage of transforming the data you received from the field into data that you can analyze.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Cleaning}}
+The cleaning process involves (1) making the data set easily usable and understandable, 
+and (2) documenting individual data points and patterns that may bias the analysis.
+The underlying data structure does not change.
+The cleaned data set should contain only the variables collected in the field.
+No modifications to data points are made at this stage, except for corrections of mistaken entries.
+
+Cleaning is probably the most time consuming of the stages discussed in this chapter.
+This is the time when you obtain an extensive understanding of  the contents and structure of the data that was collected.
+Explore your data set using tabulations, summaries, and descriptive plots.
+You should use this time to understand the types of responses collected, both within each survey question and across respondents.
+Knowing your data set well will make it possible to do analysis.
+
+
+% Deidentification ------------------------------------------------------------------
+The initial input for data cleaning is the raw data.
+It should contain only materials that are received directly from the field.
+They will invariably come in a host of file formats and nearly always contain personally-identifying information.\index{personally-identifying information}
+These files should be retained in the raw data folder \textit{exactly as they were received}.
+The folder must be encrypted if it is shared in an insecure fashion,\sidenote{\url{https://dimewiki.worldbank.org/wiki/Encryption}}
 and it must be backed up in a secure offsite location.
 Everything else can be replaced, but raw data cannot.
```

Comment:
I do not think _replaced_ is the right word here. What about _re-generated_? Or _re-created from the raw data_?

Reply:
Edited the wording in c80a095. Let me know what you think.

Reply:
Edited in c80a095:

>Every other file is created from the raw data, and therefore can be recreated.
>The exception, of course, is the raw data itself, so it should never be edited directly.